at_onboarding_flutter/lib/widgets/custom_dialog.dart

```diff
@@ -1143,6 +1147,21 @@ class _CustomDialogState extends State<CustomDialog> {
               cramSecret = await validatePerson(value, email, otp, context,
                   isConfirmation: true);
               return cramSecret;
+            } else if (value != null) {
+              AtStatusImpl atStatusImpl =
+                  AtStatusImpl(rootUrl: AppConstants.serverDomain);
+              AtStatus status = await atStatusImpl.get(value);
+
+              /// if atsign is not yet activated, it will be activated.
+              if (status.atSignStatus == AtSignStatus.unavailable &&
```

Comment:
You can remove `atSignStatus` comparison. Cause that was returning `unavailable` for any atsign. Reference [Image 1](https://cdn.discordapp.com/attachments/756903745241088011/993118732442415244/unknown.png), [Image 2](https://cdn.discordapp.com/attachments/756903745241088011/993118753845936178/unknown.png)

Reply:
Added this check to confirm atSign is in unactivated state.
Even I noticed, atSign always remains in unactivated state, looks like a bug in at_server package.

Reply:
Yes, I can take a look there